scripts/release/orthoinference/setup_orthoinference.pl

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+
+use autodie qw/:all/;
+use Cwd;
+use File::Basename;
+use File::Spec;
+use Getopt::Long;
+
+use GKB::Config;
+use GKB::CommonUtils;
+
+my ($user, $password, $database, $host, $port, $orthopairs_path, $release, $release_date, $person_id);
+GetOptions(
+    'user:s' => \$user,
+    'pass:s' => \$password,
+    'db:s' => \$database,
+    'host:s' => \$host,
+    'port:i' => \$port,
+    'orthopairs_path:s' => \$orthopairs_path,
+    'release:s' => \$release,
+    'release_date:s' => \$release_date,
+    'person_id:s' => \$person_id,
```

Comment:
config.properties path? With default set to `data-release-pipeline/orthoinference/src/main/resources/`

Reply:
Resolving since I did not realize config.properties gets built each time now